collect_app/src/main/res/values/strings.xml

```diff
@@ -653,6 +651,8 @@
     <string name="rank_items">Rank items</string>
     <string name="image_not_saved">The image can not be saved because of a problem with the corresponding instance file.</string>
     <string name="loading_form_failed">An error occurred while loading the form. Please try again.</string>
+    <string name="missing_google_account_dialog_title">Google account not set</string>
+    <string name="missing_google_account_dialog_desc">To submit forms via Google Sheets you have to setup your Google account in the Server settings.</string>
```

Comment:
Darn, just caught an issue here. "setup" is a noun. In this case, we'd like the verb form which is "set up."

But it would be even better to reuse `google_invalid_account_title` and `google_invalid_account_description`. In general, we should be really careful about only adding new strings when absolutely necessary because they add a reading burden for enumerators and a translation burden for translators.

Reply:
`because they add a reading burden for enumerators and a translation burden for translators.`
That's a really good point. Will definitely keep that in mind when creating strings.